study.md

```diff
@@ -20,21 +20,22 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+<!-- The event loop is when there functions are put into a call stack because only one function can be executed at a time. other functions are put into a queue to be executed, as the stack functions are executed the queue loops the ones that havent been called yet to the stack.-->
 ```
 
 ## Synchronicity and Asynchronicity
 
 In your own words, describe the difference between sync and async.
 
 ```md
-<!-- your answer here -->
+<!-- sync would be things running concurrent with eachother, async means that one thing at a a time runs but while we are waiting for a function to finish we can execute another function.  -->
 ```
```

Comment:
This is a little mixed up. Does this clear things up a bit? http://stackoverflow.com/questions/748175/asynchronous-vs-synchronous-execution-what-does-it-really-mean

In async we're able to execute multiple threads at once, allowing some to wait for others. I just wanted to make sure you understood that.